### PR TITLE
Add Alt tags

### PR DESCRIPTION
**Pull request by GayathriVenkatesh**
# Description
Added short alt-texts to images. 
Have wrapped the images of Readme files in `<section role="presentation">` so that they are skipped by screen readers.

## Corresponding Issue
#973 

---

Reviewing this pull request? Check out our [Code Review Practices](https://github.com/ifmeorg/ifme/wiki/Code-Review-Practices) guide if you haven't already!

### CODE REVIEW

File: app/views/pages/_contributor_blurb.html.erb [hunk 1]
```diff
@@ -12,13 +12,13 @@
     body: <<~HEREDOC
       <div class="contributorModalBody">
         <div class="contributorModalBodyPhoto hideOnSmall">
-          <img class="contributorAvatar" src="#{ProfilePicture.normalize_url(blurb["image"])}">
+          <img class="contributorAvatar" src="#{ProfilePicture.normalize_url(blurb["image"])} alt="Contributor">
```
Comment (julianguyen) [inline app/views/pages/_contributor_blurb.html.erb]: I think these avatars are not important to the context of this modal. So we can make this image purely presentational by using `alt=""`. 

Similar comment for your change below!

Comment (GayathriVenkatesh) [reply app/views/pages/_contributor_blurb.html.erb]: Sure! Will fix it.

**Comment by julianguyen** [comment]
It looks like you'll need to rebase from master! Let me know if you need any help :)

**Comment by GayathriVenkatesh** [comment]
Hi @julianguyen, I have updated the files accordingly. Please let me know if there are any issues :)

**Pull request merged**